src/main/app/Resources/modules/security/registration/store/selectors.js

```diff
@@ -35,6 +35,11 @@ const existingOrganizations = createSelector(
   (store) => store.existingOrganizations
 )
 
+const formData = createSelector(
+  [store],
+  (store) => store.form.data
```

Comment:
You must use existing form selectors to retrieve it (like this you need to know the structure of the form store).

`formSelectors.data(formSelectors.form(state, FORM_NAME))`